frontend/layouts/common/antd-override.css

```diff
@@ -67,7 +67,9 @@ a:hover {
 	border-radius: 2px;
 }
 
-.ant-btn-default:hover {
+.ant-btn-default:hover,
+.ant-btn:hover,
+.ant-btn:focus {
```

Comment:
this will also affect primary buttons e.g. Add New on dashboard, is it intended?

Reply:
yes, correct me if i'm wrong but without it, the hover and focus will have the antd default blue outline?

Reply:
previously the primary button hover is a solid darker purple, now it's white with the purple outline since it also has the `ant-btn` class

Reply:
btw if the app colors are going to be consistent we can consider customizing the theme as per [antd](https://ant.design/docs/react/customize-theme) in [nextjs](https://www.elvisduru.com/blog/how-to-customize-ant-design-theme-in-nextjs) but 1) adopter and shelter do differ and 2) need to fiddle with build options

Reply:
ah ok i missed that out.

i think they are supposed to be consistent and I asked UIUX team before but no reply. maybe we can come up with our own scheme

Reply:
since theming will be explored further in another ticket, we can go with this for now 🙂